### PR TITLE
Shift to yarnpkg

### PR DESCRIPTION
**Pull request by saneyuki**
Fix https://github.com/voyagegroup/tslint-config-fluct/issues/16

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+before_install:
+    # Use the latest yarnpkg because Travis' builtin yarn is not the latest.
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
+    - which yarn # for debugging Travis
+    - echo $PATH # for debugging Travis
+
+install:
+    - yarn install
+
+before_script:
```
Comment (nekoya) [inline .travis.yml]: Is this line needed?

**Comment by saneyuki** [comment]
@nekoya 

I fixed!

**Comment by nekoya** [comment]
@popuko r+

**Comment by popuko** [comment]
:pushpin: Commit 120b665c8ea8592aedae56f96c45d86b96b64700 has been approved by `nekoya`

**Comment by popuko** [comment]
:hourglass: 120b665c8ea8592aedae56f96c45d86b96b64700 has been merged into the auto branch 66367c472365e3f602c5386c50af281b680ff493

**Comment by popuko** [comment]
:tada: The result of what tried to merge this pull request is `success`.

* [The Travis CI build passed](https://travis-ci.org/voyagegroup/tslint-config-fluct/builds/240629620?utm_source=github_status&utm_medium=notification)

**Pull request merged**